src/gtk-4.0/widgets/adw/MessageDialog.scss

```diff
@@ -0,0 +1,25 @@
+.messagedialog {
+    padding: rem(12px);
+
+    .message-area .title-2 {
+        font-size: 1.22em;
+        font-weight: 500;
+        margin-bottom: 0.25em;
+    }
+
+    separator {
+        opacity: 0;
+    }
+
+    .response-area {
+        margin-top: rem(18px);
+
+        button {
+            @extend .raised;
+        }
+
+        separator {
+            min-width: rem(6px);
```

Comment:
This looks like it will fail with vertical layout. `AdwMessageDialog` is not always horizontal, make the labels longer in inspector.

Reply:
Thanks for the tip! fixed :)